delta-lake/common/src/main/delta-33x-40x/scala/com/nvidia/spark/rapids/delta/common/GpuDeltaParquetFileFormatBase.scala

```diff
@@ -57,6 +57,14 @@ class GpuDeltaParquetFileFormatBase(
     isCDCRead: Boolean = false
   ) extends com.nvidia.spark.rapids.delta.GpuDeltaParquetFileFormat with Logging {
 
+  if (hasTablePath) {
+    logWarning("Input Delta table has deletion vectors. Optimizations such as file splitting " +
+      "and predicate pushdown are currently not supported for this table, which can lead to " +
+      "performance issues. To avoid this, consider disabling deletion vectors on the table and " +
```

Comment:
@razajafri 's benchmarking was showing benefits, so the wording is too strong, but ok